entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityHandler.kt

```diff
@@ -296,7 +296,8 @@ class EntityHandler(
             .then(body)
             .map {
                 val contexts = checkAndGetContext(httpHeaders, it)
-                entityService.updateEntityAttribute(entityId.toUri(), attrId, it, contexts)
+                val expandedPayload = mapOf(attrId to expandJsonLdFragment(it, contexts) as Map<String, List<Any>>)
```

Comment:
`attrId` is not expanded so calling the variable `expandedPayload` is not totally correct. And I don't think it is a good idea to have a mix of compacted and expanded data inside the same structure.

Reply:
Based on your comment:
`It would be clearer and more efficient to directly expand a mapOf(attrId to it) (attrId being later expanded in partialUpdateEntityAttribute(...))`
I've understood that we can pass a mapOf(attrId to it) to the service function.
We can either:
 - expand the attrId here and pass an expandedPayload to the service function.
 - pass a reducedPayload to be expanded later.